pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```

Comment:
There is a line in this file that we can remove now, since we don't bundle calendars
```
include = ["business/data"]
```